### PR TITLE
sql-statements: update some statements

### PR DESCRIPTION
**Pull request by zimulala**
### What is changed, added or deleted? (Required)
Update the docs for some statements, such as `change column`, `create database`, `drop column`, `drop table`, `rename index`. And some statements with "alter" prefixed statements.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): pingcap/docs-cn#3300, pingcap/docs-cn#3235

### CODE REVIEW

File: sql-statements/sql-statement-drop-table.md [hunk 1]
```diff
@@ -49,6 +49,7 @@ Query OK, 0 rows affected (0.23 sec)
 ## MySQL compatibility
 
 * Dropping a table with `IF EXISTS` does not return a warning when attempting to drop a table that does not exist. [Issue #7867](https://github.com/pingcap/tidb/issues/7867)
+* Currently `RESTRICT` and `CASCADE` are only supported grammatically.
```
Comment (yikeke) [inline sql-statements/sql-statement-drop-table.md:52]:
```suggestion
* Currently `RESTRICT` and `CASCADE` are only supported syntactically.
```

File: sql-statements/sql-statement-add-column.md [hunk 2]
```diff
@@ -76,6 +72,7 @@ mysql> SELECT * FROM t1;
 * Adding multiple columns at the same time is currently not supported.
```
Comment (yikeke) [inline sql-statements/sql-statement-add-column.md:72]:
```suggestion
* Adding multiple columns at the same time in a statement is currently not supported.
```

**Comment by yikeke** [comment]
`sql-statements/sql-statement-alter-index.md` is not aligned in this PR. Do you know which PR updates https://github.com/pingcap/docs-cn/pull/3235/files#diff-c4db67744480812d582d8e0c8344c2cbR12 in docs repo? @zimulala

**Comment by zimulala** [comment]
@yikeke 
Now we haven't the file of `sql-statements/sql-statement-alter-index.md`.

**Comment by yikeke** [comment]
> sql-statements/sql-statement-alter-index.md

Got it. I found that https://github.com/pingcap/docs-cn/pull/3011/ has not been aligned in docs right now. Will find someone to update `sql-statements/sql-statement-alter-index.md` ASAP, thanks~

**Pull request merged**